### DISCUSSION TITLE
How do I ignore the "internal-package" skip reason for monorepo?

### DISCUSSION THREAD

**Discussion by songkeys** [renovatebot/renovate#8042] [Request Help] [answered]
I'm using lerna to manage my monorepo.

As stated [here](https://github.com/renovatebot/renovate/issues/1147#issuecomment-343723354):

> Normally, lerna would patch the versions in package.json for any "internal" packages it published. i.e. no need for Renovate to do anything for them because they are updated in other package.json files at the same time they're published.

But I normally use npm to publish every single internal package rather than using lerna. That's why I would like to use renovate to bump my internal packages.

But renovate skips every internal package for me. Is there a switch to disable this feature?  Thank you.

**Comment by viceice** [comment]
Can you please provide a small reproduction repo so we can check this. But i think this is the way renovate currently works.

`debug` logs would be very helpful too.

**Reply by songkeys** [reply]
Yes. This is the way renovate currently works! There's nothing wrong with it. But I just don't want my internal packages to be skipped in my case because I don't use lerna to publish my packages (but I use lerna to build and test all in CI). And that's why none of my packages' versions got bumped. Any option to turn it off?

log snippet:

```jsonc
             "packageFile": "packages/abc/package.json",
             "deps": [
               {
                 "depType": "dependencies",
                 "depName": "@my-scope/xyz",
                 "currentValue": "^1.12.0",
                 "datasource": "npm",
                 "prettyDepType": "dependency",
                 "skipReason": "internal-package", // any way not to skip?
                 "depIndex": 0,
                 "updates": []
               },
```

**Reply by viceice** [reply]
https://github.com/renovatebot/renovate/blob/5d99ff9862bac803dcea4e6ded5f3fec34abccfc/lib%2Fmanager%2Fnpm%2Fextract%2Fmonorepo.ts#L16
https://github.com/renovatebot/renovate/blob/ad52ac9b13e7fdd87fbfcedfdfbfcf44a052ae0e/lib%2Fworkers%2Frepository%2Fprocess%2Ffetch.ts#L38

Currently no way to not skip internal-packages

**Reply by songkeys** [reply]
OK. Thank you. Is there any plan to support this option? Or what should be best practice if I want all my internal packages get handled with versions?

**Reply by viceice** [reply]
Please use the search next time

**Comment by viceice** [comment, marked as answer]
see #6887
